tensorflow/python/ops/image_ops_impl.py

```diff
@@ -773,7 +773,7 @@ def pad_to_bounding_box(image, offset_height, offset_width, target_height,
 
   This op does nothing if `offset_*` is zero and the image already has size
   `target_height` by `target_width`.
-
+  
```

Comment:
Please remove all trailing whitespace

Reply:
The whitespace is not removed
![Screenshot from 2020-01-04 13-14-23](https://user-images.githubusercontent.com/323199/71771688-3982e900-2ef4-11ea-80f4-7f94a77e333a.png)

Reply:
oh sorry.